.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
```

Comment:
is circleci/config.yml used anywhere?
If it's used. It might needs more useful echo output.